woocommerce/woocommerce-ios
# Update Woo Purple color palette

Closes: #12211


## Description


Updating Woo Purple color palette based on p6riRB-8ox-p2

## Testing instructions


- Browse thru the app and check that app looks ok in general.
- Color changes are not big so it should not change app functionality a lot

## Screenshots


Change for colors (left side is old, right side is new):

![woopurple](https://github.com/woocommerce/woocommerce-ios/assets/6242034/31c7e677-209b-42c6-8e5a-dd2babf51942)

| Before      | After |
| ----------- | ----------- |
| ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-03-20 at 11 52 59](https://github.com/woocommerce/woocommerce-ios/assets/6242034/8760be95-c701-4898-b4dd-259fffd2316e)      | ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-03-20 at 11 49 38](https://github.com/woocommerce/woocommerce-ios/assets/6242034/fa2f009c-1652-4ae1-9f48-22e5c439e9fc)       |

---
